cli/src/main/resources/application.conf

```diff
@@ -22,6 +22,7 @@ msbConfig {
     # -1 means unlimited
     consumerThreadPoolQueueCapacity = 20
     requeueRejectedMessages = true
+    prefetchCount = 0
```

Comment:
Why didn't we introduce another key to switch between automatic and explicit acknowledgement? It seems like we need one.